py4DSTEM/process/dpc/dpc.py

```diff
@@ -393,7 +393,7 @@ def construct_illumation(shape, size, keV, aperture, ap_in_mrad=True,
 
     # Get diffraction space coordinates
     qsize = (float(Nx)/xsize,float(Ny)/ysize)
-    qx,qy = make_qspace_coords(shape, qsize)
+    qx, qy = np.meshgrid(np.fft.fftfreq(Nx, qsize[0]), np.fft.fftfreq(Ny, qsize[1]))
```

Comment:
I think what we want here is probably the `utils` fn `make_Fourier_coords2D`.  (It is exactly what you have here, but the fn exists and is used elsewhere, so I would call that for consistency.)